### ISSUE TITLE
Path.glob to find files not working

### ISSUE THREAD

**Issue by nachoferre** [devopshq/artifactory#62]
we are having issues with this code from you examples
````
from artifactory import ArtifactoryPath
path = ArtifactoryPath(
    "http://repo.jfrog.org/artifactory/distributions/org/")

for p in path.glob("**/*.gz"):
    print(p)
````

it throws the following error
`NotImplementedError: Non-relative patterns are unsupported`

Im having exactly the same i=error in my code please let me know whats wrong here

**Comment by Gernatch** [comment]
Same issue here.

**Comment by vertrost** [comment]
+1, v.0.4.107, python 3.7.3

**Comment by florczakraf** [comment]
I've just tried to use this lib and faced the exact same issue. Quick debugging session showed that this regression has been introduced in commit 1526b934d3132521c9f22f48972996e70477ce94 (#53, #31).

In my case (`**/*.json`) `splitroot` used to return:
```
('', '', '**/*.json')
```
with that patch it's:
```
('**', '/*.json/', '')
```
which interferes with the rest of the `pathlib` implementation. I suppose that if you don't base on anything that `0.4.107` brought, you can try rollingback to `0.4.106` to use globs until someone fixes this or the maintainer decides to revert the PR that introduced the regression.

**Comment by stac47** [comment]
@nachoferre this is implemented (and working) on the develop branch.

**Comment by allburov** [comment]
Fx available in pypi `doqh-artifactory==0.4.108`

Could someone test it?

**Comment by vertrost** [comment]
> Fx available in pypi `doqh-artifactory==0.4.108`
> 
> Could someone test it?

Thank you, works fine! Tested on corporate artifactory and on example in the first message.
